smi-conformance/grpc/handlers.go

```diff
@@ -21,20 +21,30 @@ var (
 		PortSvcB: "9091",
 		PortSvcC: "9091",
 	}
+	istioConfig = &test_gen.Linkerd{
+		PortSvcA: "9091",
+		PortSvcB: "9091",
+		PortSvcC: "9091",
+	}
 )
 
 func (s *Service) RunTest(ctx context.Context, req *conformance.Request) (*conformance.Response, error) {
 	results := make([]*conformance.SingleTestResult, 0)
 	var config test_gen.ServiceMesh
 
+	config = linkerdConfig
+	labels := make(map[string]string)
```

Comment:
@kumarabd , can we accept labels from req, i wasn't sure of that, so hardcoded atm...
Only this argument needs to be changed for future changes too...
Rest should work fine...

Reply:
@nveenjain  sure let me have a look.